lib/ui/pages/news_list/news_list_page.dart

```diff
@@ -27,46 +27,60 @@ class _NewsPageState extends State<NewsListPage> {
         child: Column(
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
-            PrimaryNewsListItem(
-              news: PrimaryNewsListEntity(
-                imageUrl: 'https://i.picsum.photos/id/9/800/800.jpg',
-                title: "New laptops seem to be stuck to the board",
-                authorName: "Marek Aureliusz",
-                authorImageUrl:
-                    'https://www.ludoviccareme.com/files/image_211_image_fr.jpg',
-              ),
+            Container(
```

Comment:
jeżeli chodziło o dodanie odstępu do góry to wystarczy SizedBox, a jeżeli cel był inny to generalnie linijka ustawiająca width może zostać usunięta i efekt będzie identyczny :)

Reply:
tu chciałem tylko sprawdzić czy wysokość horyzontalnej listy będzie się skalować w zależności od górnego odstępu i tak jest. Stworzenie poprawnego layoutu całego tego widoku to inny task

Reply:
też powinno być jakieś oznaczenie tego zamysłu